model/github.py

```diff
@@ -22,10 +22,11 @@
 from model.base import (
     BasicCredentials,
     NamedModelElement,
+    ModelValidationError,
 )
 
 
-class PreferredProtocol(enum.Enum):
+class Protocol(enum.Enum):
     SSH = 'ssh'
     HTTPS = 'https'
```

Comment:
yet another nit: the procotol is `HTTP`, even if it is used w/ transport-level encryption (TLS)